app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/useProjectManagerAdapter.ts

```diff
@@ -0,0 +1,93 @@
+/** @file Compatible with the desktop and the web client */
+import { useMemo } from 'react'
+import { ProjectManager } from 'enso-studio-content/src/project_manager'
+import * as auth from '../authentication/providers/auth'
+import { Project, ProjectState, useBackendService } from './service'
+
+interface ProjectManagerAdapter {
+    createProject(projectName: string, projectTemplateName?: string): Promise<Project>
+    listProjects(): Promise<Project[]>
+}
+
+const useProjectManagerDesktopClient = (
+    projectManager?: ProjectManager
+): ProjectManagerAdapter | null => {
+    if (!projectManager) return null
+    const { organization } = auth.useFullUserSession()
+    const baseOnEmptyProject = (
+        project: Partial<Project> & Pick<Project, 'organizationId' | 'name' | 'projectId'>
+    ): Project => ({
+        state: { type: ProjectState.created },
+        packageName: 'Main',
+        address: null,
+        ami: null,
+        ideVersion: null,
+        engineVersion: null,
+        ...project,
+    })
+    return {
+        async createProject(projectName, projectTemplateName) {
+            const createdProject = await projectManager.createProject(
```

Comment:
lint error (unsafe assignment of `any` value)

Reply:
I don't see the error